exampleCourse/questions/templateRandomMultipleChoiceNumericChoices/server.py

```diff
@@ -0,0 +1,24 @@
+import random
+
+def generate(data):
+
+    # Sample a random decimal number in the tenths place between 1.1 and 5
+    a = round(random.uniform(1.1, 5), 1)
+
+    # Sample an integer number between 6 and 11
+    b = random.randint(6, 11)
+
+    # Put these two decimal numbers into data['params']
+    data['params']['a'] = a
+    data['params']['b'] = b
+
+    # Compute the product of these two numbers
+    c = a * b
+
+    # Put the product into data['params']
+    data['params']['correct_answer'] = c
+    
+    # Generate three distractors by changing the operation
+    data['params']['wrong_answer1'] = a * (b - 1)
+    data['params']['wrong_answer2'] = b - a
+    data['params']['wrong_answer3'] = a + b
```

Comment:
This is actually a great example of how we can get accidentally generate duplicate answers. If `a = 1.2` and `b = 6` then `a * b = 7.2 = a + b`, so `wrong_answer3 = correct_answer`.

It's safer to generate distractors here with `(a - 1) * b` and `(a + 1) * b` or something like that, which must be distinct from `a * b`.

We should save this example as a good case study of how it's possible to accidentally generate bad questions.

Reply:
Whoops, I'll add an issue ticket on distractors.